web/docs/guides/auth/auth-apple.mdx

```diff
@@ -0,0 +1,218 @@
+---
+id: auth-apple
+title: "Apple Login"
+description: Setting up Apple Login
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Description
+
+How to set up Apple Login ("Sign In With Apple")
+
+
+The setup process consists of six parts:
+
+* obtain an `App Id` with “Sign In with Apple” capabilities
+* obtain a `Services Id` - this will serve as the `client_id`
+* obtain a `secret key` that will be used to get our `client_secret`
+* generate the `client_secret` using the `secret key`
+* Add your `client id` and `client secret` keys to your [Supabase Project](https://supabase.io)
+* Add the login code to your [Supabase JS Client App](https://github.com/supabase/supabase-js)
+
+## Steps
+
+### Log into your [Apple Developer Account](https://developer.apple.com)
+
+- Go to [https://developer.apple.com](https://developer.apple.com) 
+- Click on `Account` at the top right to log in
+
+Once logged into your *"Apple Deveoper Account"*:
+
+### Obtain an App Id with “Sign In with Apple” capabilities.
+
+- Go to Certificates, Identifiers & Profiles
+- Click on "Identifiers" at the left
+- Click on the + sign in the upper left next to “Identifiers”
+- Select "App IDs" and click "Continue"
+- Select type "App" and click "Continue"
+- Fill out your app information
+    - App description
+    - Bundle ID (Apple recommends reverse-domain name style, so if your domain is acme.com and your app is called roadrunner, use: "com.acme.roadrunner")
+    - Scroll down and check "Sign In With Apple"
+    - Click "Contine" at the top right
+    - Click "Register" at the top right
+
+### Obtain a Services ID
+This will serve as the `cliend_id` when you make API calls to authenticate the user.
+
+- Go to Certificates, Identifiers & Profiles
```

Comment:
Either 

Go to "Certificates, Identifiers & Profiles" or Go to `Certificates, Identifiers & Profiles`

Something to differentiate the "action text" from the "external text"